apiserver/plane/api/views/project.py

```
@@ -250,7 +250,7 @@ def post(self, request, slug):
                 serializer.save()
                 self.create_project(
                     request.META.get("HTTP_ORIGIN"), 
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)